license_solver/classifiers.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+# solver-license-job
+# Copyright(C) 2021 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Class download classifiers and extract data from them."""
+
+import re
+import sys
+import urllib.request
+
+
+def _get_abbreviation(classifier: str) -> list:
```

Comment:
You can be more specific with return types:

```suggestion
def _get_abbreviation(classifier: str) -> List[str]:
```